spec/application/navbar_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+RSpec.feature "Page has navbar", :type => :feature do
+  context 'when the user is signed in' do
+    before(:each) do
+      user = FactoryBot.create(:user)
+      sign_in user
+    end
+
+    it "has a dropdown menu" do
+      visit root_path
+      expect(page).to have_css(".navbar .dropdown")
+    end
+
+    it "dropdown menu shows menu button" do
+      visit root_path
+      expect(page).to have_css(".navbar .dropdown-toggle", text: "Menu")
```

Comment:
Better to use `t()` here I believe.

Reply:
Move this file into spec/features/...?

Reply:
`t()` didn't work as expected in the specs, But this is not needed anymore, because the drop down menu shouldn't show `Menu`, but rather the first name of the user (as you pointed out correctly)

Reply:
Ok, I think prefixing `t` with the module name worked for me (`I18n.t(...)`).